pkg/apiserver/common/common.go

```diff
@@ -67,100 +58,95 @@ func NewService(
 func Register(r *gin.RouterGroup, s *Service) {
 	endpoint := r.Group("/common")
 
-	endpoint.GET("/pods/all", s.GetPods)
-	endpoint.GET("/namespaces/all", s.GetNamespaces)
-	endpoint.GET("/kinds/all", s.GetKinds)
+	endpoint.GET("/pods", s.getPods)
+	endpoint.GET("/namespaces", s.getNamespaces)
+	endpoint.GET("/kinds", s.getKinds)
 
 }
 
-// GetPods returns the list of pods
-func (s *Service) GetPods(c *gin.Context) {
-	pods := make([]Pod, 0)
+// @Summary Get pods from Kubernetes cluster.
+// @Description Get pods from Kubernetes cluster.
+// @Tags common
+// @Produce json
+// @Param ns query string false "namespace"
+// @Success 200 {array} Pod
+// @Router /api/common/pods [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) getPods(c *gin.Context) {
+	listOptions := &client.ListOptions{}
+	ns := c.Query("ns")
```

Comment:
@cwen0 I will update this param to `namespace` to be consistent with other APIs in my pr if you think it is okay.